### DISCUSSION TITLE
How to format prior data to remove trailing slashes from URLs

### DISCUSSION THREAD

**Discussion by d19dotca** [umami-software/umami#1811] [General]
I've recently modified the Umami install to remove trailing slashes for consistency-sake, but the data still shows the trailing slashes from prior-generated data. Is there any recommended way to edit the data to remove the trailing slash at all from existing URLs in the database by any chance? Any suggestions? I know it's a little bit OCD but would love to have this be consistent across the various websites.

**Comment by briancao** [comment]
You're going to have to run SQL against your data.  Something to the effect of this post.

https://stackoverflow.com/questions/10843989/how-to-remove-leading-and-trailing-slash-in-mysql

example: 
```
-- select sample data
select url as old_url, TRIM(TRAILING '/' FROM url) as new_url from event
where url like '%/'
and url != '/'
limit 100;


-- perform update
update event
set url = TRIM(TRAILING '/' FROM url)
where url like '%/'
and url != '/'

```
